db/scripts/3_create_l2_tables.sql

```diff
@@ -0,0 +1,51 @@
+\connect rollup;
+
+
+CREATE TABLE optimistic_canonical_chain_batch (
+  id BIGSERIAL NOT NULL,
+  batch_number BIGINT NOT NULL,
+  submitted_tx_batch_l1_tx_hash character(66) DEFAULT NULL,
+  submitted_root_batch_l1_tx_hash character(66) DEFAULT NULL,
```

Comment:
No, this is meant to be 1:1. If they shouldn't be, we'll need to do some redesign. Do we have any docs that indicate how these batches may differ so that I can design around it?